appveyor.yml

```diff
@@ -11,6 +11,14 @@ init:
           Update-AppveyorBuild -Version ($env:APPVEYOR_REPO_TAG_NAME).TrimStart("v")
       }
 
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  version: '{version}'
+  package_version: '{version}'
```

Comment:
I found in the documentation that these only work if:

https://www.appveyor.com/docs/build-configuration/#net-core-csproj-files-patching

> Note that specific attribute like PackageVersion should exist in .csproj file to be patched. Note that the xmlns must not be defined or else the file won’t be processed.

So none of the attributes were patched cos they didn't exist in the csproj.